cli/cmd/describe.go

```diff
@@ -110,17 +65,17 @@ func getInstrumentationLabelTexts(workload *unstructured.Unstructured, ns *v1.Na
 	if workloadFound {
 		instrumented = workloadLabel == consts.InstrumentationEnabled
 		if instrumented {
-			decisionText = "Workload is instrumented because the " + workload.GetKind() + " contains the label '" + consts.OdigosInstrumentationLabel + "=" + workloadLabel + "'"
+			decisionText = "Workload is instrumented because the " + workload.Kind + " contains the label '" + consts.OdigosInstrumentationLabel + "=" + workloadLabel + "'"
```

Comment:
IMO, you can create a template of this text and just fill the needed values (including with/out -> NOT)

Reply:
Potentially if someone will edit this - he can forget to do it in one the places

Reply:
mmm, each line is a bit different, and this function is the only place where we manage this text, so I think it's ok. perhaps if it gets more complicated in the future, we can refactor this